_pkgdown.yml

```diff
@@ -13,13 +13,66 @@ repo:
 news:
   cran_dates: true
 
-
 navbar:
   structure:
     left: [reference, news]
   components:
     reference:
-       text: Reference
-       href: reference/index.html
+      text: Reference
+      href: reference/index.html
 
+reference:
+  - title: "Datasets"
+    desc: "List of all available datasets."
+    contents:
+      - ae
```

Comment:
We should add `@keywords dataset` to the `*.R` files (except `get_terms.R`) and use 
```
      - has_keyword("dataset")
```
here.
Then we don't need to maintain this list.